.github/workflows/bump-modules.yml

```diff
@@ -11,13 +11,19 @@ permissions:
 jobs:
   update-Modules:
     runs-on: ubuntu-22.04
+    env:
+      COMMIT_NAME: github-actions[bot] 
+      COMMIT_EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
The `[]` seem to make this an invalid email. 
![Screenshot_20240721-121949.png](https://github.com/user-attachments/assets/7b38f5cd-9a7a-494c-99cb-0e863c34aad6)



We can instead just use a generic ffmuc mail and then give the bot our own name.

Reply:
i would stick to the github actions e-mail for the profil Picture:
![image](https://github.com/user-attachments/assets/243d8840-af39-4371-b2aa-d8f0d6ae351e)
also it seems to work just fine with the `[]` also copied from the create-pull-request workflow:
https://github.com/peter-evans/create-pull-request/blob/d884c1e53f6f67d5ba26f3abc0159ad6c12c16d8/action.yml#L27

Reply:
OK, apparently they really use the `[...]`. 
I just checked and our dependabot has a different ID:
```
commit 764ef92afb5531ab81e2349d9a4d2644131eea04
Author:     dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
AuthorDate: Wed Jun 19 07:33:52 2024 +0000
Commit:     GitHub <noreply@github.com>
CommitDate: Wed Jun 19 07:33:52 2024 +0000
```

Reply:
The current ID redirects to the generic actions page :+1: